x/feegrant/keeper/keeper.go

```diff
@@ -228,3 +288,26 @@ func (k Keeper) ExportGenesis(ctx sdk.Context) (*feegrant.GenesisState, error) {
 		Allowances: grants,
 	}, err
 }
+
+func (k Keeper) removeFromGrantQueue(ctx sdk.Context, key []byte) {
+	store := ctx.KVStore(k.storeKey)
+	store.Delete(key)
+}
+
+func (k Keeper) insertAllowanceKey(ctx sdk.Context, grantKey []byte, exp *time.Time) {
+	store := ctx.KVStore(k.storeKey)
+	store.Set(feegrant.FeeAllowancePrefixQueue(exp, grantKey), grantKey)
```

Comment:
let's try to not store the value (use empty bytes: []byte{}). We can extract `grantKey` form the key.
Other solution will be to map `expireTime -> list of keys`. But then we always have to firstly check if the list is there (to append) before inserting to not overwrite.

Reply:
empty bytes sounds good.